Login

tngur1101 opened the pull request:
login markup부터 올립니다

tngur1101 committed:
login

tngur1101 committed:
Merge branch 'develop' of https://github.com/caulipse/caulipse-ui into login

tngur1101 committed:
nothing

tngur1101 committed:
pr

tngur1101 committed:
icon change

Jeyoung-Park commented on src/app/login/boxContainer/LoginBoxContainer.tsx:
```diff
@@ -0,0 +1,40 @@
+const boxcontainer = (): JSX.Element => {
+	type ID = { email : string };
+	type PW = { pw: string };
+	const [inputId, setInputId] = useState<ID | null>(null);
```
이거 굳이 객체로 관리하시는 이유 있으신가요?

Jeyoung-Park commented:
> login markup부터 올립니다

@tngur1101 
pr draft로 올리신 이유가 있으신가요? 
추가적으로 기존 pr close하고 새로 올리신 이유도 궁금합니다.

Jeyoung-Park commented on src/app/login/footerContainer/LoginFooterContainer.scss:
```diff
@@ -0,0 +1,34 @@
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.36;
```
여기 단위 없는데 확인 부탁드려요

Jeyoung-Park commented on src/app/login/loginLogo/loginlogo.tsx:
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import './LoginLogo.scss';
```
위에 있는 파일 네임보면 `./loginlogo.scss`로 해야 참조가 됐을 것 같네요. 확인 부탁드려요

Jeyoung-Park commented on src/app/signup/boxcontainer/SignupBoxContainer.scss:
```diff
@@ -0,0 +1,40 @@
+    margin: 1rem 0 10.094rem;
+    padding: 0.875rem 4.813rem;
+    background-color: #fefefe;
+    margin-left: 7rem;
```
사소한 거긴 한데 위에 있는 margin 속성이랑 같이 쓰면 더 깔끔할 것 같아요

Jeyoung-Park commented on src/app/signup/signupid/SignupId.tsx:
```diff
@@ -0,0 +1,60 @@
+    return(
+        <div>
+            <div>
+                <span className="NickName">닉네임</span>
```
여기 className 컨벤션 안 지켜진 것 같네요
확인 부탁드려요

Jeyoung-Park commented on src/app/login/footerContainer/LoginFooterContainer.scss:
```diff
@@ -0,0 +1,34 @@
+.login-footer{
```
여기 파일이름은 카멜케이스로 되어야합니다
컨벤션은 리드미 참고 부탁드려요

Jeyoung-Park commented on src/app/login/footerContainer/LoginFooterContainer.scss:
```diff
@@ -0,0 +1,34 @@
+  width: 50%;
+  height: 1.25rem;
+  margin: 2.5rem 4.563rem 0 5.625rem;
+  margin-left: 38vw;
```
이거 가운데 정렬 되야되는 거 같네요. marginleft값 이거 맞는지 확인부탁드려요~

Jeyoung-Park commented on src/app/signup/boxcontainer/SignupBoxContainer.scss:
```diff
@@ -0,0 +1,40 @@
+    border-radius: 6rem;
+    border: solid 1px var(#e2e2e2);
+    background-color: var(#fefefe);
+    margin-left: 3%;
```
마진값 퍼센트 값으로 적용하는 거 맞는지 확인 부탁드려요

Jeyoung-Park commented on src/pages/login/styles.scss:
```diff
@@ -0,0 +1,15 @@
+    margin: 0.063rem 0 5.875rem 4.563rem;
+    margin-left: 70rem;
+    margin-top: 3.563rem;
+    margin-right: 1.25rem;
```
이거 위에 있는 마진 값이랑 다 겹칠 거 같네요

tngur1101 committed:
issue

tngur1101 committed:
footer

tngur1101 committed:
onclick

tngur1101 committed:
login css

tngur1101 committed:
icon 삭제

tngur1101 committed:
signup css & link

tngur1101 committed:
css

tngur1101 committed:
span color

tngur1101 committed:
media query maxwidth

tngur1101 committed:
border color

tngur1101 committed:
input padding

tngur1101 committed:
css

tngur1101 committed:
border 2px

tngur1101 committed:
box-sizing 추가

tngur1101 committed:
button padding

tngur1101 committed:
inputhandle & css

tngur1101 committed:
login align:center

tngur1101 committed:
button css

tngur1101 committed:
signup css

tngur1101 committed:
Delete src/app/login/loginLogo directory

tngur1101 committed:
Add files via upload

taechonkim commented on src/App.tsx:
```diff
@@ -39,6 +40,7 @@ const MainContainer = (): JSX.Element => {
 					<Route exact path="/study/:category" component={StudyPage} />
 					<Route path="/profile" component={ProfilePage} />
 					<Route path="/login" component={LoginPage} />
```
@tngur1101 로그인은 모달으로 개발이 되어야 합니다. 즉 라우터에서도 빠져야합니다! cc. @Jeyoung-Park

taechonkim commented on src/app/login/boxContainer/LoginBoxContainer.scss:
```diff
@@ -0,0 +1,71 @@
+  margin-top: 3.5rem;
+  margin-bottom: 0.5rem;
+  // margin-left: 1.25rem;
+  // margin-right: 1.25rem;
```
@tngur1101 주석 처리하신 부분은 제거해주시면 좋을 것 같습니다

taechonkim commented on src/app/login/boxContainer/LoginBoxContainer.scss:
```diff
@@ -0,0 +1,71 @@
+  // margin-left: 1.25rem;
+  // margin-right: 1.25rem;
+  border-radius: 6px;
+  border: solid 2px #e2e2e2;
```
@tngur1101 제플린 시안상 인풋 필드 보더는 1px 입니다

tngur1101 committed:
Update LoginBoxContainer.scss

taechonkim commented on src/app/login/boxContainer/LoginBoxContainer.scss:
```diff
@@ -0,0 +1,71 @@
+}
+
+input:invalid {
+  border: 1px border red;
```
@tngur1101 에러상태일때 보더 컬러 red 가 아닙니다~ 디자인 확인 부탁드려요

taechonkim commented on src/app/login/boxContainer/LoginBoxContainer.scss:
```diff
@@ -0,0 +1,71 @@
+
+.login-btn{
+  display: inline-block;
+  width: 60%;
```
@tngur1101 디자인 시안상 로그인 버튼과 아이디, 패스워드 인풋 필드 width 가 동일한 것으로 보입니다 확인 부탁드립니다!

taechonkim commented on src/app/login/boxContainer/LoginBoxContainer.tsx:
```diff
@@ -0,0 +1,50 @@
+		console.log('clicked!');
+	}
+
+	return(
```
@tngur1101 로그인은 모달이기 때문에 디자인 시안상 우측 상단에 클로즈 아이콘이 있습니다. 추가 부탁드립니다!

taechonkim commented:
<img width="398" alt="스크린샷 2022-03-26 오후 9 57 50" src="https://user-images.githubusercontent.com/56914200/160240629-d4fd145b-ff7b-45e4-b99d-3ed423298c10.png">
@tngur1101 

1. 포커스 상태일 때 보더 컬러가 디자인 시안과 다름
2. 플레이스홀더 컬러 디자인 시안과 다름 
3. validation 처리 및 에러 메시지 노출 로직 추가 필요 (https://app.zeplin.io/project/60ab2c1eda632a27c275f616/screen/61def0b52ea389126541af41)

taechonkim commented on src/app/signup/boxcontainer/SignupBoxContainer.scss:
```diff
@@ -0,0 +1,64 @@
+  margin-bottom: 0.5rem;
+  padding: 1rem 10rem 1rem 0rem;
+  border-radius: 6px;
+  border: solid 2px #e2e2e2;
```
@tngur1101 여기도 보더가 1px 으로 수정도어야 합니다

taechonkim commented on src/app/signup/boxcontainer/SignupBoxContainer.scss:
```diff
@@ -0,0 +1,64 @@
+  // background-color: var(#fefefe);
+}
+
+.signup-btn{
```
@tngur1101 디자인 시안과 다른 부분이 많은 것 같습니다! (폰트 컬러나 백그라운드 컬러 등 )

taechonkim commented on src/app/signup/boxcontainer/SignupBoxContainer.scss:
```diff
@@ -0,0 +1,64 @@
+  font-weight: 300;
+}
+
+.forgotten-btn{
```
@tngur1101 백그라운ㄷ 컬러와 폰트 컬러 확인 부탁드립니다

taechonkim commented on src/app/signup/boxcontainer/SignupBoxContainer.tsx:
```diff
@@ -0,0 +1,26 @@
+	}
+
+    return(
+        <div>
```
@tngur1101 디자인 시안상 뒤로가기 버튼이 좌측 상단에 있습니다. 추가 부탁드립니다!

taechonkim commented on src/pages/signup/index.tsx:
```diff
@@ -0,0 +1,22 @@
+const SignupPage = (): JSX.Element => {
+
+    return(
+        <div className="signup-default">
```
@tngur1101 회원가입 첫번째 페이지에는 하단에 footer 가 있습니다. (개인정보 이용약관 관련) 확인 부탁드립니다!

tngur1101 committed:
Update LoginBoxContainer.tsx

tngur1101 committed:
css

css

tngur1101 committed:
input:focus border

input focus했을 때 border색이 변하도록 설정했는데 마음대로 변하지 않아서 코드 한번 봐주실수있나요?

tngur1101 committed:
input invalid border

얘도 처음에 input:invalid{
 border: solid 1px #6a1e69
}로 했는데 위에만 적용이 되어서 밑에 비밀번호는 고민을 해봐야할것같아요

tngur1101 committed:
Update index.tsx

login modal을 넣는다고 넣었는데 오류 나는데 어떻게 해야할지 잘 모르겠어요

tngur1101 committed:
Add files via upload
